Formula/gr-osmosdr.rb

```diff
@@ -0,0 +1,101 @@
+class GrOsmosdr < Formula
+  desc "Osmocom GNU Radio Blocks"
+  homepage "https://osmocom.org/projects/sdr/wiki/GrOsmoSDR"
+  url "https://cgit.osmocom.org/gr-osmosdr/snapshot/gr-osmosdr-0.1.4.tar.xz"
+  sha256 "1945d0d98fd4b600cb082970267ec2041528f13150422419cbd7febe2b622721"
+
+  head do
+    url "git://git.osmocom.org/gr-osmosdr"
+
+    if build.with?("librtlsdr")
+      opoo "Building --HEAD --with-librtlsdr requires `brew install librtlsdr --HEAD`"
```

Comment:
@tdsmith I'm hitting precisely the issue in https://github.com/Homebrew/legacy-homebrew/issues/17387#issuecomment-91925218 with the `librtlsdr` dependency.